### ISSUE TITLE
DATASETS PAGE COLOR BUTTONS

### ISSUE THREAD

**Issue by zomer-g** [OriHoch/data4dappl#76]
![image](https://user-images.githubusercontent.com/30901226/45253096-f77e7d00-b369-11e8-889a-e66202c78a8e.png)
https://www.odata.org.il/dataset

all text should be clear white #FFFFFF
"מסמכים", "ארגונים", "רשויות", "אודות" ו "הוסיפו רשות"

the background of numbers in the right side should be black #000000 instead of grey #777777

![image](https://user-images.githubusercontent.com/30901226/45253151-22b59c00-b36b-11e8-8054-c821dc0fa32b.png)

the numbers in the bottom of the page should be clear white #FFFFFF if have blue background

the blue button of "הוסיפו מסמכים" should be in #2B72AB instead of #307DBE
this should change also in organization and group pages

**Comment by OriHoch** [comment]
done + deployed
